### ISSUE TITLE
Change "from" field in the "reset password instructions" email

### ISSUE THREAD

**Issue by eddierubeiz** [sciencehistory/chf-sufia#1199]
At the [reset password page](https://digital.sciencehistory.org/users/password/new), if you enter your email you will eventually get a message from “please-change-me-at-config-initializers-devise@example.com”. This could definitely be something more friendly. Maybe "reset_password@digital.sciencehistory.org".

**Comment by jrochkind** [comment]
Good catch. Maybe we even reveal our `digital-tech@sciencehistory.org` address, to use a real address?

**Comment by eddierubeiz** [comment]
Sure!

**Comment by sanfordd** [comment]
Did a quick check, the devise.rb has the mail sender set to config.mailer_sender = "no-reply@sciencehistory.org

**Comment by jrochkind** [comment]
Yeah, I thought I remembered that actually. But @eddierubeiz you have actually tested it and found that other email address? Weird!

**Comment by eddierubeiz** [comment]
Correct - I had Mary Mark Ockerbloom forward me the e-mail she got and that was the sender email.
